nimrod/test_suite_generation/generators/test_suite_generator.py

```diff
@@ -70,6 +70,9 @@ def _compile_test_suite(self, input_jar: str, test_suite_path: str, extra_class_
         class_path = generate_classpath([input_jar, test_suite_path, compiled_classes_path, JUNIT, HAMCREST] + extra_class_path)
 
         for java_file in self._get_test_suite_class_paths(test_suite_path):
-            self._java.exec_javac(java_file, test_suite_path, None, None,
-                                  '-classpath', class_path, '-d', compiled_classes_path)
+            try:
+                self._java.exec_javac(java_file, test_suite_path, None, None,
+                                    '-classpath', class_path, '-d', compiled_classes_path)
+            except:
+                pass
```

Comment:
Considerando que conhecemos a exceção que ocorre, poderíamos levantá-la, e continuar com a execução do `for`. O que acha?

Reply:
Acrescentei uma mensagem no log